src/authenticator.ts

```diff
@@ -1,9 +1,4 @@
-import {
-  isSession,
-  redirect,
-  Session,
-  SessionStorage,
-} from "@remix-run/server-runtime";
+import { isSession, Session, SessionStorage } from "@remix-run/server-runtime";
```

Comment:
This is still using a Remix package, would it be ok for TSS to depend on this? This doesn't really depend on anything from Remix, maybe it could convince the team to move them to a separate agnostic package?

Another option could be to make Authenticator use another interface like's unstorage and provider wrappers of SessionStorage for unstorage in the Remix package and another in the TanStack package.